newIDE/app/src/EventsSheet/InstructionEditor/NewInstructionEditorDialog.js

```diff
@@ -146,6 +167,22 @@ export default function NewInstructionEditorDialog({
     }
   };
 
+  const addBehavior = (type: string, defaultName: string) => {
+    if (!chosenObject) return;
+    setNewBehaviorDialogOpen(false);
+
+    addBehaviorToObject(project, chosenObject, type, defaultName);
+
+    // Re-choose the same object to force recomputation of chosenObjectInstructionsInfoTree
+    chooseObject(chosenObject.getName());
+  };
+
+  const onAddExtension = () => {
```

Comment:
nit: onExtensionAdded or onExtensionInstalled, to show that this is something happening at the end (and not something to be called when the user wants to add an extension)